whois-api/src/main/java/net/ripe/db/whois/api/rest/SyncUpdatesService.java

```diff
@@ -191,6 +192,10 @@ private Response doSyncUpdate(final HttpServletRequest httpServletRequest, final
 
             final UpdateContext updateContext = new UpdateContext(loggerContext);
 
+            if( RestServiceHelper.isHttpProtocol(httpServletRequest) ){
+                updateContext.addGlobalMessage(UpdateMessages.httpSyncupdate());
+            }
```

Comment:
I usually use addMessage. What is the difference between addGlobalMessage vs addMessage?

Reply:
public void addMessage(final UpdateContainer updateContainer, final Message message) 
-> You specify the updateContainer so the message is related to that container (that means that is related to the updated object)

 public void addGlobalMessage(final Message message) {
-> This is not related to one specific object 

example:

- From-Host: 127.0.0.1
- Date/Time: Wed Aug 16 14:23:41 2023Z

SUMMARY OF UPDATE:

Number of objects found:                   1
Number of objects processed successfully:  1
  Create:         1
  Modify:         0
  Delete:         0
  No Operation:   0
Number of objects processed with errors:   0
  Create:         0
  Modify:         0
  Delete:         0

DETAILED EXPLANATION:

***Warning: This Syncupdates request used insecure HTTP, which may be removed in
            a future release. Please switch to HTTPS.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The following object(s) were processed SUCCESSFULLY:

---
Create SUCCEEDED: [role] FR1-TEST   Abuse Role


***Warning: Value changed due to conversion of IDN email address(es) into
            Punycode

***Warning: There are no limits on queries for ROLE objects containing
            "abuse-mailbox:"

Reply:
(example via Mattermost as well)